pkgs/top-level/python-packages.nix

```diff
@@ -10158,6 +10158,7 @@ self: super: with self; {
 
   qtconsole = callPackage ../development/python-modules/qtconsole { };
 
+  qtile-unwrapped = callPackage ../development/python-modules/qtile { };
```

Comment:
```suggestion
  qtile = callPackage ../development/python-modules/qtile { };
```

Reply:
Hmm, am I right in thinking that this change would require other changes too? For instance in [qtile/wrapper.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/window-managers/qtile/wrapper.nix) and [qtile-extras/default.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/qtile-extras/default.nix). Those files use `qtile-unwrapped`. If I change those to use `qtile` instead, will they use the top-level `qtile` or python-module `qtile`? Or can they remain unchanged due to the alias `qtile-unwrapped = python3.pkgs.qtile;`?

Reply:
> Hmm, am I right in thinking that this change would require other changes too?

Yes but those are required anyway to make this change clean.



> If I change those to use `qtile` instead, will they use the top-level `qtile` or python-module `qtile`?

toplevel because the other is python3Packages.qtile or python3.pkgs.qtile.

> Or can they remain unchanged due to the alias `qtile-unwrapped = python3.pkgs.qtile;`?

aliases in aliases.nix are forbidden inisde of nixpkgs.